src/main/resources/default-world.yml

```diff
@@ -0,0 +1,396 @@
+world_name: world
+clear_ores:
+  this_world:
```

Comment:
Just eliminate `this_world` here, assuming you made the change prior, and have it go:
```
clear_ores:
  replace:
  - etc
```